src/hevm/src/EVM/Types.hs

```diff
@@ -571,7 +559,7 @@ foldExpr f acc expr = acc <> (go expr)
         <> (go b)
         <> (go c)
         <> (go d)
-        <> (go g)
+        <> (foldl (<>) mempty (fmap go g))
```

Comment:
It'd be nice to know what the parameters to `Create` are. Right now, `a b c d g h` is a bit obscure, so the change on `g` is obscure :S Can we change `Create a b c ...` to  `Create offs sz mem logs sto addr` ? Then I'd get the change on `logs` :)

Reply:
Yeah these traversals are a pretty obscure in general, but I'm not sure how important it is to assign a distinct name to the params here, since it's more about the structure of the expressions than the actual meaning of each variable?

Reply:
Yeah, it's true that it would just be a maintenance nightmare, and it's pretty easy to look it up.